docs/conf.py

```diff
@@ -67,8 +67,9 @@
     'sphinx_tabs.tabs',
 
     # Falcon-specific extensions
-    'ext.rfc',
     'ext.doorway',
+    'ext.private_args',
```

Comment:
Do we still need this extension after refactoring in favour of the LRU approach?
(But maybe it's still useful to keep so that we don't accidentally surface private args? :thinking: )

Reply:
I suppose we can leave it in since it isn't hurting anything, but I don't have a strong preference either way.